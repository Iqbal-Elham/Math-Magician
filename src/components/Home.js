import React from 'react';
import styles from './styles/Home.module.css';

const Home = () => (
  <div className={styles.content}>
    <h2>Welcome to our Math page</h2>
    <p>
      Lorem, ipsum dolor sit amet consectetur adipisicing elit. Corporis qui
      tempora dolores ratione dignissimos nisi dolore, vel mollitia impedit
      recusandae. Commodi, omnis? Impedit eius doloribus eum tempore ullam
      placeat nostrum!Lorem, ipsum dolor sit amet consectetur adipisicing elit. Corporis qui
      tempora dolores ratione dignissimos nisi dolore, vel mollitia impedit
      recusandae. Commodi, omnis? Impedit eius doloribus eum tempore ullam
      placeat nostrum!
    </p>
  </div>
);

export default Home;
