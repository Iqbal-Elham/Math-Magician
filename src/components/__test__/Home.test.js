import renderer from 'react-test-renderer';
import Home from '../Home';

it('Render the home user interaction', () => {
  const tree = renderer.create(<Home />).toJSON();
  expect(tree).toMatchSnapshot();
});
