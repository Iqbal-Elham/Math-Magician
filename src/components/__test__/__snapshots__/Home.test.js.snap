// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot of the home page 1`] = `
<div
  className="content"
>
  <h2>
    Welcome to our Math page
  </h2>
  <p>
    Lorem, ipsum dolor sit amet consectetur adipisicing elit. Corporis qui tempora dolores ratione dignissimos nisi dolore, vel mollitia impedit recusandae. Commodi, omnis? Impedit eius doloribus eum tempore ullam placeat nostrum!Lorem, ipsum dolor sit amet consectetur adipisicing elit. Corporis qui tempora dolores ratione dignissimos nisi dolore, vel mollitia impedit recusandae. Commodi, omnis? Impedit eius doloribus eum tempore ullam placeat nostrum!
    <br />
    <br />
    Lorem, ipsum dolor sit amet consectetur adipisicing elit. Corporis qui tempora dolores ratione dignissimos nisi dolore, vel mollitia impedit recusandae. Commodi, omnis? Impedit eius doloribus eum tempore ullam placeat nostrum!Lorem, ipsum dolor sit amet consectetur adipisicing elit. Corporis qui tempora dolores ratione dignissimos nisi dolore, vel mollitia impedit recusandae. Commodi, omnis? Impedit eius doloribus eum tempore ullam
  </p>
</div>
`;
